core/src/main/java/apoc/trigger/Trigger.java

```diff
@@ -12,47 +13,42 @@
 import java.util.Map;
 import java.util.stream.Stream;
 
+import static apoc.trigger.TriggerNewProcedures.TriggerInfo;
 /**
  * @author mh
  * @since 20.09.16
  */
 
 public class Trigger {
-    public static class TriggerInfo {
-        public String name;
-        public String query;
-        public Map<String,Object> selector;
-        public Map<String, Object> params;
-        public boolean installed;
-        public boolean paused;
-
-        public TriggerInfo(String name, String query, Map<String, Object> selector, boolean installed, boolean paused) {
-            this.name = name;
-            this.query = query;
-            this.selector = selector;
-            this.installed = installed;
-            this.paused = paused;
-        }
+    // public for testing purpose
+    public static final String SYS_NON_WRITER_ERROR = "It's not possible to write into a cluster member with a non-writer role system database.\n";
```

Comment:
I think this message could perhaps be improved because it doesn't really have anything to do with the "system" database AFAIK. 

"This instance is not allowed to write to the database because it is a follower. Please use these procedures against the database leader.\n"